### PR TITLE
Add on_winner_choose hook

### PR DESCRIPTION
**Pull request by GenaMinenkov**
Related to https://github.com/splitrb/split/issues/571

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -386,6 +386,8 @@ Split.configure do |config|
   # before experiment reset or deleted
   config.on_before_experiment_reset  = -> (example) { # Do something on reset }
   config.on_before_experiment_delete = -> (experiment) { # Do something else on delete }
+  # after experiment winner had been set
+  config.on_winner_choose = -> (experiment) { # Do something on winner choose }
```
Comment (andrehjr) [inline README.md]: What do you think about renaming it to `on_experiment_winner_choose` ? (keeping all experiment hooks with the same naming pattern)

Comment (GenaMinenkov) [reply README.md]: Ok, will change.

**Comment by andrehjr** [comment]
Hi @GenaMinenkov,

Do you still want to continue this PR?

File: README.md (hunk 1 above)
Comment (GenaMinenkov) [reply README.md]: Hey @andrehjr, I have updated a hook name.

Sorry for delay 😅

Comment (andrehjr) [reply README.md]: Awesome, no worries 💖 Thanks for taking the time!

**Pull request merged**